packages/react-checkbox/src/components/Checkbox/useCheckboxStyles.ts

```diff
@@ -4,226 +4,209 @@ import { CheckboxState } from './Checkbox.types';
 
 export const checkboxClassName = 'fui-Checkbox';
 
+// TODO replace these spacing constants with theme values once they're on the theme
+const spacingHorizontalS = '8px';
+const spacingHorizontalM = '12px';
+
+// CSS variables for the indicator colors.
+// These let the colors be picked based on the :hover, etc. states on the root
+const indicatorColor = '--fui-Checkbox-indicator-color';
+const indicatorBorderColor = '--fui-Checkbox-indicator-borderColor';
+const indicatorBackgroundColor = '--fui-Checkbox-indicator-backgroundColor';
+const indicatorSize = '--fui-Checkbox-indicator-size';
+
 /**
  * Styles for the root slot
  */
-const useStyles = makeStyles({
-  root: theme => ({
-    display: 'inline-flex',
+const useRootStyles = makeStyles({
```

Comment:
@ling1726 mentioned preferring a `useStyles` pattern w/ only one `makeStyles` call on the Select package PR: https://github.com/microsoft/fluentui/pull/20944#discussion_r763725446

Would it be good to go through and a) decide if that's the pattern we're following everywhere (or if not, when not), and b) update here + other in existing controls' style hooks?

Reply:
I personally think this is a cleaner way to organize the styles. We could possibly come up with a style guide in the future, but I'm going to leave it as-is for now.

I think @khmakoto did some perf tests a while back, checking the perf of using multiple `makeStyles` calls for a single component and didn't find any detectable perf impact. I don't think those results are recorded anywhere, so I'm not 100% sure I'm remembering the details correctly.

Reply:
I agree that separate makeStyles calls allow cleaner organization and easier maintenance